datasets/fr/assemblee/crawler.py

```diff
@@ -0,0 +1,129 @@
+"""
+Crawler for members of the French National Assembly.
+"""
+
+import json
+import re
+from typing import Any, Dict, Iterator, List
+from zipfile import ZipFile
+
+from zavod import Context, Entity
+from zavod import helpers as h
+from zavod.logic.pep import categorise
+
+REGEX_PATH = re.compile(r"^json/acteur/")
+
+
+def crawl_collabos(
+    context, person: Entity, uid: str, mandat: Dict[str, Any]
+) -> Iterator[Entity]:
+    """Add staff (parliamentry collaborators) as associates."""
+    collabos = mandat.pop("collaborateurs")["collaborateur"]
+    if not isinstance(collabos, list):
+        collabos = [collabos]
+    for i, c in enumerate(collabos):
+        collabo = context.make("Person")
+        collabo.id = context.make_slug(uid, f"collabo{i}")
+        h.apply_name(
+            collabo,
+            prefix=c.pop("qualite"),
+            first_name=c.pop("prenom"),
+            last_name=c.pop("nom"),
```

Comment:
fail - this comment was intended for `context.audit_data(c, ["dateDebut", "dateFin"])`

Reply:
ah ok! yes these are usually empty, but just in case they aren't...

Incidentally a nice improvement (but it might not be easy to implement) to `parse_date` would be for it to allow partial matching, e.g. if you have `%Y-%m-%d` as a format but the date is "2024-03-06 08:15:00" then it would give you the full date instead of just "2024".  `strptime` most definitely does not do this, unfortunately.